<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/airsim/Neighborhood/drone_1cam_lidar_noisedimu_202011220319.bag"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pointcloud_to_depthimage)/rviz_config/pointcloud_to_depthimage_airsim.rviz"/>

	<!-- NED to NEU -->
	<node pkg="msg_conversion" name="pc_ned_to_neu" type="pc_ned_to_neu">
		<remap from="/cloud" to="/airsim_node/drone/lidar/LidarCustom"/>
	</node>
	<!-- <node pkg="pc_store" name="lower_pc_hz" type="lower_pc_hz"> -->
	<!-- 	<remap from="/cloud" to="/cloud/neu"/> -->
	<!-- 	<param name="target_hz" type="double" value="20"/> -->
	<!-- </node> -->

	<!-- MAIN -->
	<node pkg="pointcloud_to_depthimage" name="pointcloud_to_depthimage" type="pointcloud_to_depthimage" output="screen">
		<!-- <remap from="/cloud" to="/cloud/lower_hz"/> -->
		<remap from="/cloud" to="/cloud/neu"/>
		<!-- <param name="num_ring" type="int" value="32"/> -->
		<!-- <param name="points_per_ring" type="int" value="1092"/> <!&#45;&#45; HDL&#38;#45;32E &#45;&#45;> -->
		<param name="points_per_ring" type="int" value="1812"/>	<!-- VLP&#45;32C -->
		<!-- <param name="fov_upper_deg" type="double" value="15.0"/> -->
		<!-- <param name="fov_lower_deg" type="double" value="&#45;25.0"/> -->
		<!-- <param name="max_range" type="double" value="100.0"/> -->
	</node>
</launch>
